### DISCUSSION TITLE
Buat yang mac gimana bang executablenya?

### DISCUSSION THREAD

**Discussion by Stackopoy** [Arya-f4/worldshellfinder#7] [Q&A] [answered]
mohon dijawab bang 🙏

**Comment by Aryma-f4** [comment, marked as answer]
Buat yang mac itu udah saya coba di laptop temen ane bisa bang, tapi build sendiri. Pas aku coba build sendiri di laptop windows ku ga bisa bang, makanya ga aku cantumin.
